docs/user-manual/en/configuration-index.md

```diff
@@ -112,7 +112,7 @@ system-property-prefix | Prefix for replacing configuration settings using Bean
 [transaction-timeout](transaction-config.md "Resource Manager Configuration")              |  how long (in ms) before a transaction can be removed from the resource manager after create time. Default=300000
 [transaction-timeout-scan-period](transaction-config.md "Resource Manager Configuration")  |  how often (in ms) to scan for timeout transactions. Default=1000
 [wild-card-routing-enabled](wildcard-routing.md "Routing Messages With Wild Cards")        |  true means that the server supports wild card routing. Default=true
-[network-check-NIC](network-isolation.md) | The network internet card to be used on InetAddress.isReacheable
+[network-check-NIC](network-isolation.md) | The network internet controller to be used on InetAddress.isReacheable
```

Comment:
NIC is Network Interface Controller. I had to google that and I guess it won't be obvious to more people. Maybe write the abbreviation and the meaning in parens after it, so that people learn this right away.

Also, it may not be entirely obvious what are the NIC names. Something like eth0 on Linux? What it is on Windows? How can I list them? Any alternatives to this for older systems, that do not have the more robust "enp0s25" style naming and eth0 and eth1 may be randomly switched; or is that no longer a problem?